data/json/mapgen/deserter_city/deserter_city_office.json

```diff
@@ -0,0 +1,318 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_1f",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "s|ssssssssssssssssssssss",
+        "u######VGGVGGVGGV#######",
+        "s#n???..............caY#",
+        "s#?.................c..#",
+        "s#?.n..sssss.sssss..c..#",
+        "s#?.n.s~~~~ss~~~~s..c..#",
+        "s#?...s~~~~ss~~~~s..cc.#",
+        "s#O...s~~~~ss~~~~s.....#",
+        "s#h.O.s~~~~ss~~~~s.....#",
+        "s#^...sssss.sssss......#",
+        "s#h..O.................#",
+        "s#OO..O....DaF^........#",
+        "s#######..#####VGGVVVVV#",
+        "s#T#T#T#..#'''#.........",
+        "s#+#+#+#..#'''-.........",
+        "s#iiiii+..#'''#.........",
+        "s#CSCSC#..#####.........",
+        "s#######..+iyi#.........",
+        "s#CSCSC#..#####.........",
+        "s#iiiii+..#EEE#........#",
+        "s#+#+#+#..#EEE-....#####",
+        "s#T#T#T#.^#EEH#....+..<#",
```

Comment:
It seems this elevator is represented on this level only.

Reply:
My thinking with that was that the elevator would only _be_ on one level, i.e. the rest of the levels should be empty elevator shaft. I'm happy to change it if that's not the standard, however.

I did notice when looking it over that I accidentally used t_open_air in the elevator instead of t_open_air_rooved, so I'm fixing that for now.